app/jobs/patient_list_download_job.rb

```diff
@@ -15,12 +15,7 @@ def perform(recipient_email, model_type, params, with_medication_history: false,
       raise ArgumentError, "unknown model_type #{model_type.inspect}"
     end
 
-    patients =
-      if with_exclusions
-        Patient.excluding_dead.where(id: model.assigned_patients)
-      else
-        model.assigned_patients
-      end
+    patients = model.assigned_patients.excluding_dead
```

Comment:
This is slightly different from the existing implementation `Patient.excluding_dead.where(id: model.assigned_patients)`.

Are the two queries the same? Are there performance implications?

Reply:
The newer query makes for simpler SQL.
```
> puts f.assigned_patients.excluding_dead.to_sql
SELECT "patients".* FROM "patients"
WHERE "patients"."deleted_at" IS NULL
AND "patients"."assigned_facility_id" = '10949769-acb5-4786-8c5c-9109f5f3f509'
AND "patients"."status" != 'dead'


> puts Patient.excluding_dead.where(id: f.assigned_patients).to_sql
SELECT "patients".* FROM "patients" 
WHERE "patients"."deleted_at" IS NULL
AND "patients"."status" != 'dead'
AND "patients"."id" IN (SELECT "patients"."id" FROM "patients" WHERE "patients"."deleted_at" IS NULL AND "patients"."assigned_facility_id" = '10949769-acb5-4786-8c5c-9109f5f3f509')
```

It's also a tad faster.
```
[12] pry(main)> Benchmark.ms { f.assigned_patients.excluding_dead }
=> 0.486999997519888
[13] pry(main)> Benchmark.ms { Patient.excluding_dead.where(id: f.assigned_patients) }
=> 0.8010000019567087
```

Reply:
Suite.